src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -62,7 +62,7 @@ public static final class Arm {
   }
 
   private TalonFX m_arm;
-  private DutyCycleEncoder m_encoder;
+  private static DutyCycleEncoder m_encoder;
```

Comment:
This should not be static. Also, this will be completely rewritten anyways. You shouldn't need the ArmSubsystem for PizzaboxSubsystem to work